packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -52,6 +55,19 @@ function addVNodeToChildLWC(vnode: VCustomElement) {
     ArrayPush.call(getVMBeingRendered()!.velements, vnode);
 }
 
+// [s]coped [s]lot [f]actory
+function ssf(factory: (value: any) => VNodes, slotName: string): VScopedSlotFragment {
```

Comment:
The slot (`s` function) accepts the slot name as the first argument, I would recommend doing the same here. It would also make it easier to read the generated code.